src/utils/notifications.ts

```diff
@@ -0,0 +1,12 @@
+import { NetworkHealth } from 'interfaces/networkHealth.interface'
+import slack from '../services/slack.service'
+
+export function networkHealthNotification(networkHealth: NetworkHealth) {
```

Comment:
nitpick(blocking): Use a verb for functions. This looks almost like a class.

Reply:
suggestion(non-blocking): Consider returning a `boolean` that the notification was sent.